build.gradle.kts

```diff
@@ -20,6 +21,9 @@ repositories {
 }
 
 dependencies {
+  api(project(":protocol"))
```

Comment:
we needed these to actually get all 3 subprojects when we pull in `tbdex` as a dependency